recipes/rdp/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "rdp" %}
+{% set version = "0.8" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/rdp-{{ version }}.tar.gz
+  sha256: 19ec92f0803587891a00b439441e5e23fc9410500983f02941e4cf9ac3113e4b
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - numpy
+    - python >=2.7
```

Comment:
You should use the same constraint on host and run but note that we no longer support 2.7-3.5 so, even though the package may work on 2.7 we recommend you to use >=3.6

Reply:
Fixed on https://github.com/conda-forge/staged-recipes/pull/14404/commits/e322388b16c566c5c407ea26af3a0e8281c81a99!